product-web-app/controller/bandController.ts

```diff
@@ -0,0 +1,22 @@
+import { Application, Request, Response } from 'express';
+import addBand from '../service/bandService.js';
+import Band from '../model/band.js';
+
+export default function bandController(app: Application) {
+  app.get('/admin/add-band', async (req: Request, res: Response) => {
+    res.render('add-band');
+  });
+
+  app.post('/admin/add-band', async (req: Request, res: Response) => {
+    const data: Band = req.body;
+
+    try {
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      const response = await addBand(data);
+    } catch (e) {
+      // eslint-disable-next-line no-console
+      console.error(e);
```

Comment:
do not disable it, fix it
```suggestion
    try {
      // eslint-disable-next-line @typescript-eslint/no-unused-vars
      await addBand(data);
    } catch (e) {
      // eslint-disable-next-line no-console
      console.error(e);
```